openstack/run_packstack_docker.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+set -e
+
+docker run -d --rm --name packstack  \
+-v /lib/modules/:/lib/modules/ \
+--privileged --cap-add ALL --security-opt seccomp=unconfined   --cgroup-parent=docker.slice \
+-p 0.0.0.0:5000:5000 \
+-p 0.0.0.0:8774:8774 \
+-p 0.0.0.0:8775:8775 \
+-p 0.0.0.0:8778:8778 \
+-p 0.0.0.0:9292:9292 \
+-p 0.0.0.0:9696:9696 \
+quay.io/eslutsky/packstack:latest
+
+sleep 300
+
+
+source openstack/utils.sh
+run_command healthcheck
+openstack_container network list
+run_command bash /create-cirros.sh
```

Comment:
am i missing the script  ?

Reply:
this script is in packaged in the packer container [0]. it can be  be mounted externally of you want...

[0] https://github.com/eslutsky/packstack-docker/blob/main/files/create-cirros.sh

Reply:
Thanks 👍